static/styles/right-sidebar.css

```diff
@@ -197,7 +197,39 @@
     margin-right: 10px;
 }
 
+.input-append {
+    margin-bottom: 5px;
+    font-size: 90%;
+    letter-spacing: -.3em;
+}
```

Comment:
So as a general rule, we're trying to keep our CSS organized, where things that could impact a large part of the site are in different files from things that are for just one element.  So either this should be a `#user-list .input-append` rule, or it should be moved to `static/js/components.css`...